templates/tutorial-description-template.yml

```diff
@@ -9,3 +9,9 @@
     ksql: <KSQL-ENABLED>
     kstreams: <KSTREAMS-ENABLED>
     kafka: disabled
+    c: disabled
```

Comment:
Same thing here as above. 

We'll need to update the tools scripts to accommodate different languages.  I know that's beyond the scope of this PR, I'm just putting a note here.